src/Searchable.php

```diff
@@ -118,6 +114,22 @@ public function toSearchableArray()
     {
         $array = $this->toArray();
 
+        foreach ($this->getArrayableRelations() as $key => $value) {
```

Comment:
I've done this backwards, sorry haha but here too.

```suggestion
        foreach ($this->getArrayableRelations() as $key => $model) {
```